src/DetailView/EventDiagram.js

```diff
@@ -1,12 +1,104 @@
 import React, { Component } from 'react';
 import { css } from 'aphrodite';
+import plotly from 'plotly.js/dist/plotly.min';
+import Utils from './../Utils/Utils';
 
 class EventDiagram extends Component {
+
+	static sortEventsByTime(eventA, eventB, timeStampAttributeName) {
+		const dateA = new Date(EventDiagram.getTimeStampAsStringFromEvent(eventA, timeStampAttributeName));
+		const dateB = new Date(EventDiagram.getTimeStampAsStringFromEvent(eventB, timeStampAttributeName));
+		return dateA - dateB;
+	}
+
+	static getTimeStampAsStringFromEvent(event, timeStampAttributeName) {
+		return event.Attributes.find(attribute =>
```

Comment:
Is ensured, that `find` will find something? If not you'll try to receive `.Value` of undefined.